src/os/portable/os-impl-posix-files.c

```diff
@@ -198,11 +198,15 @@ int32 OS_FileChmod_Impl(const char *local_path, uint32 access)
     struct stat st;
     int         fd;
 
-    /* Open file to avoid filename race potential */
+     /* Open file to avoid filename race potential */
```

Comment:
Errant whitespace change?

Reply:
looks like it, I will fix it when I get the inevitable conflict with #656